packages/core/admin/admin/src/content-manager/history/components/VersionDetails.tsx

```diff
@@ -3,40 +3,79 @@ import * as React from 'react';
 import { ContentLayout, HeaderLayout, Main, Typography } from '@strapi/design-system';
 import { Link } from '@strapi/design-system/v2';
 import { ArrowLeft } from '@strapi/icons';
+import { Contracts } from '@strapi/plugin-content-manager/_internal/shared';
 import { useIntl } from 'react-intl';
-import { NavLink, useNavigate } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
 
-const VersionDetails = () => {
+/* -------------------------------------------------------------------------------------------------
+ * VersionHeader
+ * -----------------------------------------------------------------------------------------------*/
+
+interface VersionHeaderProps {
+  headerId: string;
+  version: Contracts.HistoryVersions.HistoryVersionDataResponse;
+}
+
+const VersionHeader = ({ headerId, version }: VersionHeaderProps) => {
   const { formatMessage } = useIntl();
-  const navigate = useNavigate();
+
+  return (
+    <HeaderLayout
+      id={headerId}
+      title={`History version ${version.id}`}
```

Comment:
Shouldn't this be translated? 🤔

Reply:
Not really because it's just a placeholder, after we implement the next ticket the content will be different